app/build.gradle.kts

```diff
@@ -1,44 +1,35 @@
+import uk.gov.authentication.config.ApkConfig
+
 plugins {
-    alias(libs.plugins.android.library)
-    alias(libs.plugins.android.kotlin)
-    alias(libs.plugins.detekt)
-    alias(libs.plugins.ktlint)
-    alias(libs.plugins.kotlin.serlialization)
-    id("maven-publish")
-    id("authentication.jvm-toolchains")
-    id("sonarqube-module-config")
-    id("jacoco")
-    id("jacoco-module-config")
+    `maven-publish`
+    id("uk.gov.authentication.android-lib-config")
 }
 
-apply(from = "${rootProject.extra["configDir"]}/detekt/config.gradle")
-apply(from = "${rootProject.extra["configDir"]}/ktlint/config.gradle")
-
 android {
-    namespace = "${rootProject.extra["baseNamespace"]}.pages"
-    compileSdk = (rootProject.extra["compileAndroidVersion"] as Int)
-
     defaultConfig {
-        minSdk = (rootProject.extra["minAndroidVersion"] as Int)
-
-        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-        consumerProguardFiles("consumer-rules.pro")
+        namespace = ApkConfig.APPLICATION_ID + ".impl"
+        compileSdk = ApkConfig.COMPILE_SDK_VERSION
+        minSdk = ApkConfig.MINIMUM_SDK_VERSION
+        targetSdk = ApkConfig.TARGET_SDK_VERSION
```

Comment:
As in, we don't need to specify the target SDK?

I've basically copied this configuration from the Logging repository so that it's consistent:
https://github.com/govuk-one-login/mobile-android-logging/blob/main/modules/logging-api/build.gradle.kts

Reply:
yeah, think the tagertSdk is optional, but doesn't hurt